components/apimgt/org.wso2.carbon.apimgt.internal.service/src/main/java/org/wso2/carbon/apimgt/internal/service/impl/ApisApiServiceImpl.java

```diff
@@ -50,30 +50,30 @@
     private static final Log log = LogFactory.getLog(ApisApiServiceImpl.class);
 
     @Override
-    public Response apisGet(String xWSO2Tenant, String apiId, String context, String version, String gatewayLabel, String accept,
-                            MessageContext messageContext) throws APIManagementException {
-
+    public Response apisGet(String xWSO2Tenant, String apiId, String context, String version, String gatewayLabel,
+            Boolean expand, String accept, MessageContext messageContext) throws APIManagementException {
```

Comment:
When `expand` is not sent, the default "true" doesn't seem to set from the code generation layer.. 
Shall we verify it and set it manually from here?

Reply:
tested the behavior. ApisApi.java generated the annotation to set this default value